be/src/io/output_stream.h

```diff
@@ -0,0 +1,43 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2022-present, StarRocks Limited.
+
+#pragma once
+
+#include "io/writable.h"
+
+namespace starrocks::io {
+
+class OutputStream : public Writable {
+public:
+    struct Buffer {
```

Comment:
Maybe we can use Slice, or put the definition of Buffer outside.

Reply:
updated